{"name": "test_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'dict' object has no attribute 'access_token'", "trace": "client = <api_client.APIClient object at 0x000001EDA2F43020>\n\n    def test_login(client):\n        response = client.login(\n            email=CREDENTIALS[\"email\"],\n            password=CREDENTIALS[\"password\"],\n            client_name=CREDENTIALS[\"client_name\"],\n        )\n    \n>       print(response.access_token)\nE       AttributeError: 'dict' object has no attribute 'access_token'\n\ntest_login.py:18: AttributeError"}, "start": 1737026098106, "stop": 1737026099115, "uuid": "edd4c1a5-a9c5-4281-9b11-5f346a6606ff", "historyId": "edf0f70bc2e89eec38a945fd0769f980", "testCaseId": "edf0f70bc2e89eec38a945fd0769f980", "fullName": "tests.API.test_login#test_login", "labels": [{"name": "parentSuite", "value": "tests.API"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DESKTOP-HRHMBRL"}, {"name": "thread", "value": "23756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.API.test_login"}]}